### ISSUE TITLE
Crash with a cryptic error message when a folder container a dot

### ISSUE THREAD

**Issue by thib-rdr** [miickel/gulp-angular-templatecache#145]
Hi all,

When using `angularTemplatecache()` to concatenate all my templates to a template.js file, it crashes when a folder with a dot is present in the project.

`
TypeError: Cannot read property 'toString' of null
    at templateCacheFile (path/node_modules/gulp-angular-templatecache/index.js:89:36)
    at wrappedMapper (path/node_modules/event-stream/node_modules/map-stream/index.js:84:19)
`
Thanks !

**Comment by simonua** [comment]
Hi @thib-rdr, is this issue still relevant? If so, could you create a PR to address this, please? If not, could you close the issue, please?

**Comment by ashclarke** [comment]
I recreated this problem by accident:

```
components
|
--- news-item-title.tpl.html
     |
     --- news-item-title.tpl.html
```

I'd accidentally named my folder with the same convention, and in this case exactly the same name, as my template files.

**Comment by simonua** [comment]
@ashclarke, thanks for the repro. I'm trying to weigh how much of a use-case a folder with a dot may be. It's perhaps not unusual but I'm struggling to determine whether it's more than an edge case. I'd love to review a PR if you have time and inclination to dive further into it.

**Comment by ashclarke** [comment]
No worries. Wouldn’t mind looking into into - might be able to have a bash
at the weekend.

**Comment by simonua** [comment]
@ashclarke, that would be very appreciated. No rush. Just happy to hear willingness to contribute. If you do end up getting into it, could you please add appropriate tests, too?

Thank you very much!

**Comment by simonua** [comment]
Fixed by PR #171.
